### PR TITLE
TFIDInfoWriter by default subscribes to DISTSUBTIMEFRAME/0xccdb

### PR DESCRIPTION
**Pull request by shahor02**

### CODE REVIEW

File: Detectors/GlobalTrackingWorkflow/src/tfidinfo-writer-workflow.cxx [hunk 1]
```diff
@@ -19,7 +19,7 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
 {
   // option allowing to set parameters
   std::vector<ConfigParamSpec> options{
-    ConfigParamSpec{"dataspec", VariantType::String, "tfidinfo:FLP/DISTSUBTIMEFRAME/0", {"spec from which the TFIDInfo will be extracted"}},
+    ConfigParamSpec{"dataspec", VariantType::String, "tfidinfo:FLP/DISTSUBTIMEFRAME/0xccdb", {"spec from which the TFIDInfo will be extracted"}},
```
Comment (sawenzel) [inline Detectors/GlobalTrackingWorkflow/src/tfidinfo-writer-workflow.cxx:22]: This does not solve the problem for running `$O2DPG_ROOT/DATA/production/configurations/2021/OCT/apass3/async_pass.sh` for me.

I am getting:
`[FATAL] error while setting up workflow in o2-tfidinfo-writer-workflow: Parse error: Expected a number`

Comment (sawenzel) [reply Detectors/GlobalTrackingWorkflow/src/tfidinfo-writer-workflow.cxx:22]: @chiarazampolli @shahor02

Comment (ktf) [reply Detectors/GlobalTrackingWorkflow/src/tfidinfo-writer-workflow.cxx:22]: Needs to be decimal. I will add support for parsing hex...

Comment (ktf) [reply Detectors/GlobalTrackingWorkflow/src/tfidinfo-writer-workflow.cxx:22]: #8298 should add support for 0xccdb (assuming Ruben is using DataDescriptorQueryBuilder::parse).

**Pull request merged**